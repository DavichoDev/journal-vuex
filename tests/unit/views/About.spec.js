import { shallowMount } from '@vue/test-utils'
import AboutPage from '@/views/About.vue'

describe('About.vue tests', () => {

    let wrapper

    beforeEach(() => {
      wrapper = shallowMount( AboutPage )

      jest.clearAllMocks()
    })

    test('Debe de renderizar el componente correctamente', () => {
        expect( wrapper.html() ).toMatchSnapshot()
    })

})