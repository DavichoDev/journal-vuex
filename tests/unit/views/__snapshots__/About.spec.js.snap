// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About.vue tests Debe de renderizar el componente correctamente 1`] = `
<div class="about">
  <h1>This is an about page</h1>
</div>
`;
